presidio-anonymizer/README.MD

```diff
@@ -5,24 +5,17 @@
 The Presidio anonymizer is a Python based module for anonymizing detected PII text
 entities with desired values.
 
+[comment]: <shiranr> (#TODO need to fill this once the the functionality is done 2656 - 2659)
 Presidio anonymizer comes by default with the following anonymizers:
 
-- Replace - replaces PII with desired value
-- Redact - removes PII completely from text
+- Replace - replaces PII with desired value <br>
+  Parameters: "new_value" - replaces existing text with this value. <br>
+  If "new_value" is not supplied default behavior: <entity_type>
```

Comment:
this line is not that clear

Reply:
which one?

Reply:
If "new_value" is not supplied default behavior: <entity_type>

Reply:
added an example hope it helps